dev/prepare_release_issue.py

```diff
@@ -289,6 +289,9 @@ def generate_issue_content(
                     console.print(f"[red]The PR #{pr_number} could not be found[/]")
                     continue
 
+            if pr.user.login == "dependabot":
+                console.print(f"[yellow]Skipping PR #{pr_number} as it was created by dependabot[/]")
+                continue
             # Ignore doc-only and skipped PRs
             label_names = [label.name for label in pr.labels]
             if "type:doc-only" in label_names or "changelog:skip" in label_names:
```

Comment:
Wonder if we should also add here skipping `type:misc/internal`

Reply:
Might be a good idea. It's not too often. You can also remove specific PR with ``--excluded-pr-list`` (which I often did after looking at some generated  issue report) -  but removing some of them manually based on type/area is a good idea.